actions/views/channel_sidebar.test.ts

```diff
@@ -29,5 +34,159 @@ describe('setCategoryCollapsed', () => {
         store.dispatch(Actions.setCategoryCollapsed(category1, false));
 
         expect(isCategoryCollapsedFromStorage(getPrefix(store.getState()), store.getState().storage.storage, category1)).toBe(false);
+    }, 10000);
```

Comment:
What is `10000` for?

Reply:
Good catch. I had some issues with the tests before I figured out the store wasn't passing along the initial state, and I thought waiting for a bit might fix it. I'll remove this.